lib/livebook_web/live/session_live.ex

```diff
@@ -872,11 +913,14 @@ defmodule LivebookWeb.SessionLive do
   end
 
   def handle_event("erase_outputs", %{}, socket) do
+    assert_policy!(socket, :edit)
```

Comment:
@jonatanklosko should this fall under execute?

Reply:
Good question, my initial though was that it impacts the notebook structure (especially when persisting outputs), but it's actually the same for evaluating stuff, so I think you're right.

Reply:
Fix it to `:execute`!